pkg/util/validation.go

```diff
@@ -18,19 +18,33 @@ package util
 
 import (
 	"regexp"
-	"strings"
 )
 
-// IsDNSLabel tests for a string that conforms to the definition of a label in
-// DNS (RFC 1123).
-func IsDNSLabel(value string) bool {
-	return IsDNS1123Label(value)
+const kubeChar string = "[A-Za-z0-9]"
+const extendedKubeChar string = "[-A-Za-z0-9_.]"
+const qualifiedToken string = "(" + kubeChar + extendedKubeChar + "*)?" + kubeChar
```

Comment:
minor: the "qualified" in "qualified name" means it has the prefix.  That makes the name awkward here.  I'd prefer qnameCharFmt, qnameExtCharFmt, and qnameTokenFmt as generic regex variable names